integrations/ollama/tests/test_chat_generator.py

```diff
@@ -565,6 +565,17 @@ def test_run_with_chat_history(self):
             city.lower() in response["replies"][-1].text.lower() for city in ["Manchester", "Birmingham", "Glasgow"]
         )
 
+    @pytest.mark.integration
+    def test_live_run_with_thinking(self):
+        chat_generator = OllamaChatGenerator(model="qwen3:0.6b", think=True)
+
+        message = ChatMessage.from_user("How many times does the letter 'r' appear in the word 'strawberry'?")
+        response = chat_generator.run([message])
+
+        assert isinstance(response, dict)
+        assert isinstance(response["replies"], list)
+        assert "thinking" in response["replies"][0]._meta
+
```

Comment:
let's also add a simple unit test (`test_run_with_thinking`), similar to `test_run`